api/turing/cluster/servicebuilder/fluentd_test.go

```diff
@@ -75,6 +75,13 @@ func TestNewFluentdService(t *testing.T) {
 				Namespace:   project.Name,
 				AccessModes: []string{"ReadWriteOnce"},
 				Size:        volSize,
+				Labels: map[string]string{
+					"app":          "test-svc",
+					"environment":  "",
```

Comment:
Could you leave a comment here on why the environment label value is empty ? Will be useful in the future.